test/WebJobs.Script.Tests/Workers/Rpc/WebHostRpcWorkerChannelManagerTests.cs

```diff
@@ -176,6 +176,8 @@ public async Task SpecializeAsync_ReadOnly_KeepsProcessAlive(string runtime, str
             IRpcWorkerChannel workerChannel = CreateTestChannel(languageWorkerName);
 
             await _rpcWorkerChannelManager.SpecializeAsync();
+            // Wait for debouce task to start
+            await Task.Delay(6000);
```

Comment:
Instead of this, can you use a TestHelpers.Await and return true whenever the event is found? That eliminates races in tests -- and could speed these up too actually.

Reply:
thanks! updated.